_build/css/main.css

```diff
@@ -373,4 +373,26 @@ table>tbody+tbody{
 
 .container33 .text {
   flex-grow: 1;
+}
+
+.sandbox-dataset .sandbox-dataset-title {
+  color: black;
+}
+
+.sandbox-dataset .image-card {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
Was this somewhere discussed? In the design images are aligned "start".

Reply:
should be fixed in https://github.com/sentinel-hub/public-collections/pull/114